modules/ROOT/pages/_partials/generic-saving-a-config-file.adoc

```diff
@@ -0,0 +1,44 @@
+include::{partialsdir}/attributes.adoc[]
+
+= Saving the {product-name} configuration file
```

Comment:
Technically this not reflecting what we do here. How about: `Obtaining the mobile SDK configuration file`

Reply:
I'll make the update